roles/bootstrap-os/tasks/bootstrap-oraclelinux.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Download Oracle Linux public yum repo
+  get_url:
+    url: "{{ repo_url }}"
+    dest: "{{ repo_dest }}"
+  vars:
+    repo_dest: /etc/yum.repos.d/public-yum-ol7.repo
+    repo_url: https://yum.oracle.com/public-yum-ol7.repo
+
+- name: Enable Oracle Linux repo
+  ini_file:
+    dest: /etc/yum.repos.d/public-yum-ol7.repo
+    section: "{{ item }}"
+    option: enabled
+    value: 1
+  with_items:
+    - ol7_latest
+    - ol7_addons
+    - ol7_developer_EPEL
```

Comment:
Why do you need EPEL?

Reply:
For installing python-httplib2 which is one of requirements in kubernetes/preinstall/default/main.yml. varible common_required_pkgs.